### PR TITLE
Feature/profile tooltips

### PR DESCRIPTION
**Pull request by m-the-hoff**
## Overview

Adds tooltips when hovering over the various profiles in the Editor Inspector for the MixedRealityToolkit prefab.

### CODE REVIEW

File: Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs [hunk 1]
```diff
@@ -78,6 +78,19 @@ public class MixedRealityToolkitConfigurationProfileInspector : BaseMixedReality
             "Editor",
         };
 
+        private static readonly string[] ProfileTabTooltips = {
+            "Settings that are global to the entire application. This includes desired room scale and coordinate offset.",
+            "Settings for choosing the camera provider, clipping, default background color and other camera specific behaviors.",
+            "Settings for Input Providers, including pointers, input actions, controllers, gestures and speech.",
+            "Settings for the visualization of the boundary floor, celing, walls, play area and tracked area.",
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs:85]:
```suggestion
            "Settings for the visualization of the boundary floor, ceiling, walls, play area and tracked area.",
```

File: Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs [hunk 2]
```diff
@@ -78,6 +78,19 @@ public class MixedRealityToolkitConfigurationProfileInspector : BaseMixedReality
             "Editor",
         };
 
+        private static readonly string[] ProfileTabTooltips = {
+            "Settings that are global to the entire application. This includes desired room scale and coordinate offset.",
+            "Settings for choosing the camera provider, clipping, default background color and other camera specific behaviors.",
+            "Settings for Input Providers, including pointers, input actions, controllers, gestures and speech.",
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs:84]:
```suggestion
            "Settings for input providers, including pointers, input actions, controllers, gestures and speech.",
```

File: Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs [hunk 3]
```diff
@@ -78,6 +78,19 @@ public class MixedRealityToolkitConfigurationProfileInspector : BaseMixedReality
...
+            "Settings for the visualization of the boundary floor, celing, walls, play area and tracked area.",
+            "Settings for the teleportation system.",
+            "Settings for the spatial mesh observer, including refresh rate, level of detail, material and physics related to the mesh.",
+            "Settings for the diagnostics heads up display.",
+            "Settings for the Scene System.",
+            "Settings for extensions, including the registry for adding new Service Providers.",
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs:90]:
```suggestion
            "Settings for extensions, including the registry for adding new service providers.",
```

**Review by keveleigh — COMMENTED** [review]
Mostly nit-level capitalization suggestions, along with a typo fix

File: Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs [hunk 4]
```diff
@@ -490,8 +503,16 @@ public override void OnInspectorGUI()
             EditorGUILayout.BeginVertical(EditorStyles.helpBox, GUILayout.Width(100));
             GUI.enabled = true; // Force enable so we can view profile defaults
 
+            GUIContent[] profileTitles = new GUIContent[ProfileTabTitles.Length];
+            for( int  profileIndex = 0; profileIndex < ProfileTabTitles.Length; profileIndex++)
```
Comment (RogPodge) [inline Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs:507]:
```suggestion
            for (int  profileIndex = 0; profileIndex < ProfileTabTitles.Length; profileIndex++)
```

Spacing nit

**Review by RogPodge — APPROVED** [review]

**Review by keveleigh — APPROVED** [review]

**Pull request merged**